File: examples/input/templates/struct.kt
package {#1}
// #BEGIN_IMPORT
// #IMPORT1 import {#IMPORT_1}
// #END_IMPORT
import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

// {#COMMENT}
@Serializable
@Parcelize
data class {#NAME}(
// #BEGIN_VAR
    // #VAR {#TYPE_PREFIX}
    // #VAR @SerialName(value = "{#VARNAME_JSON}")
    // #VAR var {#VARNAME_CAMEL}: {#TYPE},
    // #VAR {#TYPE_SUFFIX}
// #END_VAR
) : {#IMPLEMENT}Parcelable

File: examples/input/templates/struct.py
// #BEGIN_IMPORT
import json
// #IMPORT1 import {#IMPORT_1}
// #IMPORT2 from {#IMPORT_1} import {#IMPORT_2}
// #END_IMPORT

// #BEGIN_VAR
# {#COMMENT}
class {#NAME}({#IMPLEMENT}):
  // #VAR {#VARNAME_SNAKE}: {#TYPE} = {#TYPE_INIT}
// #END_VAR

  def __init__(self, data):
    self = json.loads(data)
// #BEGIN_FUNCTIONS
  // #FUNCTIONS {#FUNCTIONS}
// #END_FUNCTIONS
